tests/tracing/export/test_inference_table_exporter.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+
+from mlflow.entities import LiveSpan
+from mlflow.tracing.constant import TRACE_SCHEMA_VERSION_KEY
+from mlflow.tracing.export.inference_table import (
+    _TRACE_BUFFER,
+    InferenceTableSpanExporter,
+    get_completed_trace,
+)
+from mlflow.tracing.trace_manager import InMemoryTraceManager
+from mlflow.tracing.utils import encode_span_id, encode_trace_id
+
+from tests.tracing.helper import create_mock_otel_span, create_test_trace_info
+
+_TRACE_ID = 12345
+_REQUEST_ID = f"tr-{_TRACE_ID}"
+
```

Comment:
Should we also test the buffer handling of the TTLCache? (size, not time, that is)

Reply:
Absolutely!